tests/pytest/core/test_session.py

```diff
@@ -51,26 +51,21 @@ def test_did_default(app_request, mocker):
     assert did != "None"
 
 
-@pytest.mark.django_db
-def test_eligibility_default(app_request):
-    assert session.eligibility(app_request) is None
-
-
 @pytest.mark.django_db
 def test_eligibile_False(app_request):
     agency = models.TransitAgency.objects.first()
-    eligibility = []
+    eligibility = False
 
-    session.update(app_request, agency=agency, eligibility_types=eligibility)
+    session.update(app_request, agency=agency, eligible=eligibility)
 
     assert not session.eligible(app_request)
 
 
 @pytest.mark.django_db
 def test_eligibile_True(model_TransitAgency, app_request):
-    eligibility = model_TransitAgency.eligibility_types.first()
+    eligibility = True
```

Comment:
`eligible`

Reply:
👍 made the change [here](https://github.com/cal-itp/benefits/blob/903f76f7bf932cf401dba9e78887ee6b2da5c1b6/tests/pytest/core/test_session.py#L66)